west.yml

```diff
@@ -102,7 +102,7 @@ manifest:
       revision: 1c4fdba512b268033a4cf926bddd323866c3261a
       path: tools/net-tools
     - name: hal_nxp
-      revision: 68bbdbdec1153355b14386d8f095058a496f66f4
+      revision: pull/59/head
```

Comment:
The hal PR has been merged, so you can now update this to the new SHA

Reply:
> The hal PR has been merged, so you can now update this to the new SHA

Thank you, I will make this change.